src/lib.rs

```diff
@@ -82,13 +82,13 @@
 //!
 //! ```toml
 //! [dependencies]
-//! bio = "*"
+//! bio = "1"
 //! ```
 //!
-//! and import the crate from your source code:
+//! Now Rust-Bio modules can be used directly in your source code, for example:
 //!
 //! ```rust
-//! extern crate bio;
```

Comment:
Good find- though it looks like the repo has a number of instances of this pattern. Would you feel comfortable replacing or deleting, as appropriate, those as well?

Reply:
Sure, sounds good! Those are remainders from the early days of Rust. Rust-bio development started in 2015.